_oasis

```diff
@@ -19,7 +19,7 @@ Flag cli
 
 Test test
   Run$:               flag(tests)
-  Command:            $test_mustache
+  Command:            $test_mustache && $spec_mustache
```

Comment:
You need a flag :) Have a look at the cli flag for example. You should define something simliar.